server/room.py

```diff
@@ -40,9 +40,20 @@ def add_player_to_lobby(self, game_id, player_name):
         if not game_context:
             return None
         new_player = Player(player_name)
-        game_context.lobby.players.append(new_player)
+        game_context.lobby.add_player(new_player)
         return new_player.id
 
+    def remove_player_from_lobby(self, game_id, player_id):
+        '''
+        @param self: the room object.
+        @param game_id: the ID of the game to remove the player from.
+        @param player: the Player object to be removed.
```

Comment:
Looks like this last param is actually the player_id rather than the player. Probably a stale comment?